proteus-h2020/proteic
# Fix a bug that caused static charts to call keepDrawing

#### What's this PR do?
Fixes a bug that caused static charts to start the drawing loop when the user changed tabs.

#### Where should the reviewer start?
Chart.ts

#### How should this be manually tested?
Open a static chart, change tabs, return to the chart and check that it's not redrawing.

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)

---
> Thank you! :heart:

:rocket:
